src/runtime/process.rs

```diff
@@ -562,6 +562,7 @@ impl Default for LinuxSchedulerFlag {
 pub struct ExecCPUAffinity {
     #[serde(
         default,
+        rename = "initial",
```

Comment:
How about changing the field name?

Reply:
Thank you for the confirmation.

I think the field names should be changed as follows:

- `initial`
- `cpu_affinity_final` (with a rename)

This is because `final` is a reserved keyword in Rust.

Does this look good to you?

Reply:
> This is because final is a reserved keyword in Rust.

I didn't know that. Thanks. It looks good to me.

Reply:
Done. I also fixed the formatting errors.